bindings/python/examples/exchange/4_listen_events.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+# This example listens to the NewOutput event.
+
+from iota_sdk import Wallet, SyncOptions
+from dotenv import load_dotenv
+import json
+import os
+import time
+
+# This example uses secrets in environment variables for simplicity which should not be done in production.
+load_dotenv()
+
+if 'WALLET_DB_PATH' not in os.environ:
+    raise Exception(".env WALLET_DB_PATH is undefined, see .env.example")
+
+wallet = Wallet(os.environ.get('WALLET_DB_PATH'))
+
+account = wallet.get_account('Alice')
+
+received_event = False
+
```

Comment:
```suggestion
```

Reply:
Python formatter is doing that because below is a function definition